modules/virt-creating-vm-instancetype.adoc

```diff
@@ -40,13 +40,16 @@ endif::[]
 .Procedure
 
 . In the web console, navigate to *Virtualization* -> *Catalog* and click the *InstanceTypes* tab.
-. Select a bootable volume.
+. Select either of the following options:
+* Select a bootable volume.
 +
 [NOTE]
 ====
-The volume table only lists volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
+The bootable volume table lists only those volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
 ====
-.. Optional: Click the star icon to designate a bootable volume as a favorite. Starred bootable volumes appear first in the volume list.
+... Optional: Click the star icon to designate a bootable volume as a favorite. Starred bootable volumes appear first in the volume list.
+
+* Click *Add volume* to upload a new volume or use an existing PersistentVolumeClaim (PVC), VolumeSnapshot, or DataSource. Then click *Save*.
```

Comment:
@ShellyKa13, Regarding your comment (above) "In the clone page I think you should add an option to choose one of the vm snapshots as a source," is that for UX or me? I see that you think we should doc the cloning from (if I follow correctly) the VirtualMachine details page rather than the Snapshot tab, but in neither case is there an option shown in the new design doc to choose a VM snapshot as the source.

Reply:
Yeah the comment of what I think should have been done in the UX should be addressed to the UX team and not the doc that describes what is currently the possible options..